setup.py

```diff
@@ -4,52 +4,31 @@
 # Copyright (c) 2019 VMware, Inc. All Rights Reserved.
 # SPDX-License-Identifier: BSD-2-Clause
 
-from setuptools import setup, find_packages
+import os
+import sys
 
 from tern import Version
+from setuptools.command.install import install
+from setuptools import setup
 
 
-def _read_long_desc():
-    with open("README.md") as fp:
-        return fp.read()
+class VerifyVersion(install):
+    """Run a custom verify command"""
+    description = "Verify that the git tag matches current release version."
 
-
-def _get_requirements():
-
-    with open("requirements.txt") as fp:
-        return [requirement for requirement in fp]
+    def run(self):
+        tag = os.getenv('CIRCLE_TAG')
+        if tag != Version:
```

Comment:
Actually, the tag is of the form `v<major>.<minor>.<patch>` while `Version` is `<major>.<minor>.<patch>` so we would have to strip out the `v` first before doing a comparison.

Reply:
stripped the v from tag so it will properly compare to 'Version.' Nice catch!